templates/templates/.github/workflows/fogg_ci.yml.tmpl

```diff
@@ -108,6 +108,5 @@ jobs:
       - name: Init TFLint
         run: tflint --init
       - name: tflint
-        working-directory: {{`${{matrix.tfmodule}}`}}
         run: |
-          tflint --config ${GITHUB_WORKSPACE}/.tflint.hcl
```

Comment:
also changing to run tflint from repo root with target dir passed in as argument